src/js/api.js

```diff
@@ -97,32 +101,83 @@ export class API {
 
   getServiceRequests() {
     const url = `${this.baseUrl}/api/service-requests/`;
-    return $.get(url);
+    return $.get({
+      url,
+      headers: {
+        authorization: `Bearer ${getAccessToken()}`,
+      },
+    });
   }
 
   getServiceRequest(serviceRequestId) {
     const url = `${this.baseUrl}/api/service-requests/${serviceRequestId}`;
-    return $.get(url);
+    return $.get({
+      url,
+      headers: {
+        authorization: `Bearer ${getAccessToken()}`,
+      },
+    });
   }
 
   submitServiceRequest(serviceRequestDetails) {
     const url = `${this.baseUrl}/api/service-requests/`;
-    return $.post({ url, data: serviceRequestDetails });
+    return $.post({
+      url,
+      data: serviceRequestDetails,
+      headers: {
+        authorization: `Bearer ${getAccessToken()}`,
+      },
+    });
+  }
+
+  getServiceRequestFile(serviceRequestId, fileId, onReceivedCallback) {
+    // We use XMLHttpRequest to be able to receive a blob from the binary response
+    // see: https://developer.mozilla.org/en-US/docs/Web/API/XMLHttpRequest/Sending_and_Receiving_Binary_Data
+    const url = `${this.baseUrl}/api/service-requests/${serviceRequestId}/attachments/${fileId}/`;
+
+    let oReq = new XMLHttpRequest();
+    oReq.open("GET", url, true);
+    oReq.responseType = "blob";
+    oReq.setRequestHeader("authorization", `Bearer ${getAccessToken()}`);
+
+    oReq.onload = function () {
+      onReceivedCallback(oReq.response, fileId);
+    };
+
+    oReq.send();
```

Comment:
Super not a deal-breaker but, if we want to remain in the jQuery space, we can use https://api.jquery.com/jQuery.ajaxSetup/#jQuery-ajaxSetup-options to set up the custom bits here, and then do a `$.get` as we do in other places in the api.

Reply:
I agree it would be nicer to stay within jQuery, but I initially tried the `ajaxSetup` route but could not get it to work so I opted for the current solution.